common/test/acceptance/pages/lms/video/video.py

```diff
@@ -566,21 +568,36 @@ def select_transcript_format(self, transcript_format):
             bool: Selection Result.
 
         """
-        button_selector = self.get_element_selector(VIDEO_MENUS['transcript-format'])
+        button_selector = self.get_element_selector(VIDEO_MENUS['transcript-format']) # .video-tracks .button-more.has-dropdown
```

Comment:
The variable descriptions at the top of the file were unclear and I was constantly scrolling back up to see what they were. I decided to note them beside. Since this is done, I can clean it up.